tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -239,7 +252,8 @@ case class FormattedQualificationSummaryInfo(
     taskSpeedupFactor: Double,
     endDurationEstimated: Boolean,
     unSupportedExecs: String,
-    unSupportedExprs: String)
+    unSupportedExprs: String,
+    clusterTags: Map[String, String])
```

Comment:
so one thing we have perhaps let lapse a bit is the UI.  If we want this in UI we should either modify or file a follow up.

Reply:
Filed follow on issue https://github.com/NVIDIA/spark-rapids/issues/6650  to print to UI and add integration tests on databricks runs.